forge-gui/res/cardsfolder/a/agrus_kos_eternal_soldier.txt

```diff
@@ -4,6 +4,6 @@ Types:Legendary Creature Spirit Soldier
 PT:3/4
 K:Vigilance
 T:Mode$ BecomesTarget | ValidTarget$ Card.Self | ValidSource$ Ability.numTargets EQ1 | TriggerZones$ Battlefield | Execute$ TrigCopy | TriggerDescription$ Whenever CARDNAME becomes the target of an ability that targets only it, you may pay {1}{R/W}. If you do, copy that ability for each other creature you control that ability could target. Each copy targets a different one of those creatures. ({R/W} can be paid with either {R} or {W}.)
-SVar:TrigCopy:AB$ CopySpellAbility | Cost$ 1 RW | Defined$ TriggeredStackInstance | CopyForEachCanTarget$ Creature.YouCtrl+Other
+SVar:TrigCopy:AB$ CopySpellAbility | Cost$ 1 RW | Defined$ TriggeredStackInstance | CopyForEachCanTarget$ Creature.YouCtrl
```

Comment:
should this have the new property too?

Reply:
All these abilities are worded "for each other" so the engine already prevents that and I removed it for consistency